CHANGELOG_PENDING.md

```diff
@@ -8,6 +8,9 @@
 - Add errors when hanging invalid fields off of resource options.
   [#211](https://github.com/pulumi/pulumi-yaml/pull/211)
 
+- Add `Fn::FromBase64`
+  [#218](https://github.com/pulumi/pulumi-yaml/pull/218)
```

Comment:
Add to README.md as well and document the UTF-8 requirement there?

Reply:
on that topic opened
- #235